synapse/federation/sender/__init__.py

```diff
@@ -184,14 +183,18 @@
     "Total number of PDUs queued for sending across all destinations",
 )
 
-# Time (in s) after Synapse's startup that we will begin to wake up destinations
-# that have catch-up outstanding.
-CATCH_UP_STARTUP_DELAY_SEC = 15
+# Time (in s) to wait before trying to wake up destinations that have
+# catch-up outstanding. This will also be the delay applied at startup
+# before trying the same.
+# Please note that rate limiting still applies, so while the loop is
+# executed every X seconds the destinations may not be wake up because
+# they are being rate limited following previous attempt failures.
+WAKEUP_RETRY_PERIOD_SEC = 60
```

Comment:
We also need to update the big description at the top of this file on how all of this works now:

https://github.com/matrix-org/synapse/blob/f6321e386c804017f04c7bdfddda6afb618702ba/synapse/federation/sender/__init__.py#L108-L125

Reply:
Good point, thanks !
Done.